### ISSUE TITLE
ValueError: too many values to unpack (expected 2) with dabl.SimpleClassifier().fit()

### ISSUE THREAD

**Issue by gojira** [dabl/dabl#281]
`sc = dabl.SimpleClassifier().fit(X_train, y_train)`

results in:

```
ValueError                                Traceback (most recent call last)
<ipython-input-1-40278cfa23fa> in <module>
      4 X, y = load_digits(return_X_y=True)
      5 X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=1)
----> 6 sc = dabl.SimpleClassifier().fit(X_train, y_train)

~/miniconda3/envs/dabl/lib/python3.8/site-packages/dabl/models.py in fit(self, X, y, target_col)
    256         """
    257         self._rank_scoring = "recall_macro"
--> 258         return self._fit(X=X, y=y, target_col=target_col)
    259 
    260 

~/miniconda3/envs/dabl/lib/python3.8/site-packages/dabl/models.py in _fit(self, X, y, target_col)
    149         for est in estimators:
    150             set_random_state(est, self.random_state)
--> 151             scorers, _ = _check_multimetric_scoring(est, self.scoring_)
    152             scores = self._evaluate_one(est, data_preproc, scorers)
    153             # make scoring configurable

ValueError: too many values to unpack (expected 2)
```

```
(dabl.__version__, sklearn.__version__)
('0.1.9', '0.24.1')
```

**Comment by amueller** [comment]
Can you try updating please? You should be able to install dabl 0.2.0 from pypi now :)

**Comment by gojira** [comment]
Fixed in 0.2.0!  Sorry for taking so long!  auto sklearn is cool!
